.github/workflows/security.yml

```diff
@@ -55,6 +57,8 @@ jobs:
     needs: fossa-scan
     runs-on: ubuntu-latest
 
+    if: ${{ github.event.pull_request.head.repo.full_name == github.repository }}
```

Comment:
I'm pretty sure you dont need this, because since it **needs** fossa-scan, and it was skipped, this will be skipped too.